src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflow.cs

```diff
@@ -23,16 +22,14 @@
 using Microsoft.VisualStudio.R.Package.Repl;
 
 namespace Microsoft.R.Components.InteractiveWorkflow.Implementation {
-    public sealed class RInteractiveWorkflow : IRInteractiveWorkflow {
+    public sealed class RInteractiveWorkflow : IRInteractiveWorkflowVisual {
```

Comment:
Sure. The problem is in `IInteractiveWindowVisualComponent` which references `IWpfTextView`, `ITextBuffer` and `IInteractiveWindow`. I could technically make first two into new `IEditorView` and `IEditorBuffer` but generally UI-related stuff is not x-plat anyway. So question is how to decouple. In CM, PM, Workflow and plots interfaces there x-plat parts that manage infrastructure and UI parts.